reference/test/preconditioner/batch_isai_kernels.cpp

```diff
@@ -319,91 +334,281 @@ TYPED_TEST_SUITE(BatchIsai, gko::test::ValueTypes);
 TYPED_TEST(BatchIsai, GeneralBatchIsaiGenerationIsEquivalentToUnbatchedWithSpy1)
 {
     this->test_batch_isai_generation_is_eqvt_to_unbatched(
-        1, std::string("general"));
+        1, std::string("general"), false);
 }
 
+
+// NOTE: Reference Batched Isai (batchcsr iterative solve) takes a lot of time
+// TYPED_TEST(BatchIsai,
```

Comment:
Please create an issue so that we can track this.

Reply:
This was happening due to a bug in the reference batch gmres solver, which is now fixed, so this issue is no longer there. However, there's another problem that sometimes the iterative solver does not converge, leading to isai output getting screwed up.